__tests__/comments.test.js

```diff
@@ -45,12 +60,13 @@ describe("9. GET /api/reviews/:review_id/comments tests", () => {
               expect(res.text).toBe("invalid id");
             });
         });
-        test.skip("404: should return 404 if review_id is the correct format but is not a review_id in DB", () => {
+     
+        test("404: should return 404 if review_id is the correct format but is not a review_id in DB", () => {
           return request(app)
             .get("/api/reviews/1006/comments")
             .expect(404)
             .then((res) => {
-              expect(res.text).toBe("invalid id");
+              expect(res.text).toBe("This review has no comments");
```

Comment:
I'd quite like you to specify whether the review has no comments, or just doesn't exist. I see what you're getting at, they're both "technically" true, but what if the user then goes to look for the review with no comments and sometimes can find it, sometimes can't? Either way, it'll return an empty array from PG - so you'll need to do some reshuffling to check the circumstance.